agent/job_runner.go

```diff
@@ -326,6 +319,12 @@ func NewJobRunner(l logger.Logger, scope *metrics.Scope, ag *api.AgentRegisterRe
 			ClientCount: containerCount,
```

Comment:
Happy to get this k8s stuff outta here, but probably best to do that when me remove the "experiment".